internal/vshard/orchestrator/failover.go

```diff
@@ -154,7 +154,9 @@ func (f *failover) checkAndRecover(ctx context.Context, analysis *ReplicationAna
 	recvFunc, desc := f.getCheckAndRecoveryFunc(analysis.State)
 	if recvFunc == nil {
 		if desc != "" {
-			logger.Warn().Msg(desc)
+			logger.Warn().
```

Comment:
Check 165 line, I suppose you'd like to print dead followers there too.

Reply:
fixed